cmd/cert.go

```diff
@@ -162,34 +175,41 @@ func runCert(c *cli.Context) error {
 
 	derBytes, err := x509.CreateCertificate(rand.Reader, &template, &template, publicKey(priv), priv)
 	if err != nil {
-		log.Fatalf("Failed to create certificate: %v", err)
+		// log.Fatalf("Failed to create certificate: %v", err)
+		return err
 	}
 
-	certOut, err := os.Create("cert.pem")
+	certOut, err := os.Create(c.String("out"))
```

Comment:
We should avoid introducing any unnecessary change during a large refactoring PR

Reply:
Since the changes are there and there are tests, it's fine to keep them IMO.